velox/exec/tests/CMakeLists.txt

```diff
@@ -186,7 +194,96 @@ target_link_libraries(
   gflags::gflags
   glog::glog
   fmt::fmt
-  ${FILESYSTEM})
+  ${FILESYSTEM}
+  grpc::grpc++
+  grpc::grpc
+  gpr::gpr
+  protobuf::libprotobuf
+  z::z
+  re2::re2
+  address_sorting::address_sorting
+  cares::cares
+  upb::upb
+  boringssl::ssl
+  boringssl::crypto
+  utf8::utf8_range
+  utf8::utf8_validity
+  utf8::utf8_range_lib
+  absl::absl_kernel_timeout_internal
```

Comment:
Is protobuf or re2 pulling in absl?

Reply:
In my test environment, the protobuf installed by grpc pulls in absl.